python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -1346,7 +1328,7 @@ def _get_param_block(opt_op):
 
         for key in opt_op.input_names:
             new_shape = None
-            if key in ["Param", "Grad", "LearningRate", "Moment", "MeanSquare"]:
+            if key in ["Param", "Grad", "LearningRate"]:
```

Comment:
glad to see these are gone. Can you briefly explain what it was added and why we don't need it anymore?

Reply:
Well, it was some careless change in a very early time: https://github.com/PaddlePaddle/Paddle/pull/12798/files#diff-3a23ecb6a98b666c84525f3f6fcd2bb6L1212

the if condition lost the rmsprop op.